src/components/editor/Editor.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import * as selectors from '../../reducers/selectors';
```

Comment:
We can use the shortcut here: `import * as selectors from '@/reducers/selectors';`